### PR TITLE
make it compile with latest tidb

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: tikv/cop_handler.go [hunk 1]
```diff
@@ -67,8 +69,8 @@ func (svr *Server) handleCopDAGRequest(reqCtx *requestCtx, req *coprocessor.Requ
 		chunks = appendRow(chunks, data, rowCnt)
 		rowCnt++
 	}
-	warnings := dagCtx.evalCtx.sc.GetWarnings()
-	return buildResp(chunks, e.Counts(), err, warnings, time.Since(startTime))
+	//warnings := dagCtx.evalCtx.sc.GetWarnings()
+	return buildResp(chunks, e.Counts(), err, nil, time.Since(startTime))
```
Comment (coocood) [inline tikv/cop_handler.go]: We can change the `buildResp` `warnings` arg type as
```
func buildResp(chunks []tipb.Chunk, counts []int64, err error, warnings []stmtctx.SQLWarn, dur time.Duration) *coprocessor.Response {
```

Comment (coocood) [reply tikv/cop_handler.go]: and change append warnings as
```
selResp.Warnings = append(selResp.Warnings, toPBError(warnings[i].Err))
```

Comment (ngaut) [reply tikv/cop_handler.go]: Thank you for your comments. Addressed.

File: tikv/cop_handler.go [hunk 2]
```diff
@@ -40,6 +40,7 @@ type dagContext struct {
 func (svr *Server) handleCopDAGRequest(reqCtx *requestCtx, req *coprocessor.Request) *coprocessor.Response {
 	startTime := time.Now()
 	resp := &coprocessor.Response{}
+	// TODO: handle warning
```
Comment (coocood) [inline tikv/cop_handler.go]: The `TODO` can be removed?

**Pull request merged**